Assets/Scripts/SensitivityController.cs

```diff
@@ -0,0 +1,53 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class SensitivityController : MonoBehaviour
+{
+    [Header("UI Elements")]
+    [SerializeField] private Slider movementSlider;
+    [SerializeField] private Slider rotationSlider;
```

Comment:
privateフィールドの変数名の頭にはアンダーバーをつけよう。

```suggestion
    [SerializeField] private Slider _movementSlider;
    [SerializeField] private Slider _rotationSlider;
```